apps/doji-frontend/src/pages/schedule-session/index.tsx

```diff
@@ -0,0 +1,268 @@
+    resolver: yupResolver(createScheduleValidation),
+    defaultValues: {
+      date: new Date(),
+      startTime: new Date(),
+      endTime: new Date(),
+      participants: [],
+    },
+  })
+  const watchAll = watch()
+  useEffect(() => {
+    const param = new URLSearchParams(window.location.search)
+    scheduleSessionData.expertUsername = param.get('expert_username')
+    scheduleSessionData.serviceName = param.get('service_name')
+    const url =
+      'http://localhost:3333/api/session/service/' +
+      param.get('expert_username') +
+      '/' +
+      param.get('service_name')
+    axios.get(url).then((res) => {
+      setServiceData(res.data)
+    })
+  }, [])
+  function handleOpenDialog() {
+    setOpenDialog(true)
+  }
+
+  function handleCloseDialog(value) {
+    setOpenDialog(false)
+    setConfirm(value)
+    if (value) {
+      console.log(scheduleSessionData)
+    }
+  }
+  function selectedTagsHandler(items) {
+    console.log(items)
+  }
+  function calculateTotal() {
+    const timeDiff = watchAll.endTime.getTime() - watchAll.startTime.getTime()
+    const duration = Math.round((timeDiff * 10) / 36e5) / 10
+    return duration * serviceData.fee * (watchAll.participants.length + 1)
+  }
+  const onSubmit: SubmitHandler<scheduleModel> = async (data) => {
+    handleOpenDialog()
+    const timeDiff = data.endTime.getTime() - data.startTime.getTime()
+    scheduleSessionData.duration = Math.round((timeDiff * 10) / 36e5) / 10
+    const startDate = new Date(data.date)
+    startDate.setHours(
+      data.startTime.getHours(),
+      data.startTime.getMinutes(),
+      data.startTime.getSeconds(),
+    )
+    scheduleSessionData.startTime = startDate
+    scheduleSessionData.fee =
+      scheduleSessionData.duration * serviceData.fee * (data.participants.length + 1)
+    scheduleSessionData.participantsUsername = data.participants.map((element) => {
+      return element.value
+    })
+  }
+  if (serviceData === null) {
+    return null
+  }
+  return (
+    <Box display="flex" flexDirection="column" position="relative" minHeight="sm">
+      <TopBar title="New session" action={TopBarActionType.Back}></TopBar>
+      <Box>
+        <form onSubmit={handleSubmit(onSubmit)}>
+          <Grid container alignItems="center">
+            <Grid item xs={12}>
+              <Typography fontWeight={700} variant="title3">
+                {serviceData.title}
+              </Typography>
+            </Grid>
+            <Grid item xs={8} marginBottom={2}>
+              <Tables
+                content={'by ' + serviceData.firstname + ' ' + serviceData.lastname}
+                avatar={<Avatar></Avatar>}
+              ></Tables>
+            </Grid>
+            <Grid item xs={4} marginBottom={2} textAlign="right">
+              <Typography variant="large" fontWeight={700} color="#367D7F">
+                {serviceData.fee}
+              </Typography>
+              <Typography variant="regular" fontWeight={400} color="#367D7F">
```

Comment:
no hard-code color. I dont know what is this color but if you want to use color from our palette. you can do something like this

```tsx
              <Typography variant="regular" fontWeight={400} color="ink.darker">
```